csi.proto

```diff
@@ -628,30 +628,51 @@ message ValidateVolumeCapabilitiesRequest {
   // The ID of the volume to check. This field is REQUIRED.
   string volume_id = 1;
 
+  // Attributes of the volume to check. This field is OPTIONAL and MUST
+  // match the attributes of the Volume identified by `volume_id`.
+  map<string, string> volume_attributes = 2;
+
   // The capabilities that the CO wants to check for the volume. This
   // call SHALL return "supported" only if all the volume capabilities
   // specified below are supported. This field is REQUIRED.
-  repeated VolumeCapability volume_capabilities = 2;
+  repeated VolumeCapability volume_capabilities = 3;
 
-  // Attributes of the volume to check. This field is OPTIONAL and MUST
-  // match the attributes of the Volume identified by `volume_id`.
-  map<string, string> volume_attributes = 3;
+  // See CreateVolumeRequest.parameters.
+  // This field is OPTIONAL.
+  map<string, string> parameters = 4;
```

Comment:
This should be optional? Explicitly should not error

Reply:
already says `This field is OPTIONAL.` Is more needed?